### PR TITLE
Initial Game Logic

### PR DESCRIPTION
**Pull request by nicholasmonaco**
Created basic structures of Card and CardStack.
Began work on Game, only initializes a new game at the moment.
Added an ImageLoader class, which would be used to hold image values. It can be replaced or modified however, I just used the BufferedImages in it to write game logic.

### CODE REVIEW

File: src/main/java/solitaire/graphics/GameDisplay.java [hunk 1]
```diff
@@ -3,10 +3,10 @@
 import solitaire.game.Game;
 
 import javax.swing.*;
+import java.awt.*;
```
Comment (Blackop778) [inline src/main/java/solitaire/graphics/GameDisplay.java]: This isn't needed

File: src/main/java/solitaire/game/Card.java [hunk 2]
```diff
@@ -1,17 +1,55 @@
 package solitaire.game;
 
+import com.sun.xml.internal.ws.api.ResourceLoader;
```
[the remark this answers is not in the document]
Comment (nicholasmonaco) [reply src/main/java/solitaire/game/Card.java]: I don't think so either, it was probably IntelliJ auto-importing it.

File: src/main/java/solitaire/game/CardStack.java [hunk 3]
```diff
@@ -1,17 +1,72 @@
...
+    private int flipType;
+    private int tieredXOffset; // There are the value(s) that the cards will be offset by when they are tiered.
+    private int tieredYOffset;
+
+
+    public CardStack(int flipType){
+        this.cards = new Stack<>();
+
+        this.flipType = flipType;
+        solveFlipType(flipType);
```
[the remark this answers is not in the document]
Comment (nicholasmonaco) [reply src/main/java/solitaire/game/CardStack.java]: `cards` is empty by default, but is populated using the `Game` class

File: src/main/java/solitaire/graphics/GameDisplay.java (hunk 1 above)
Comment (nicholasmonaco) [reply src/main/java/solitaire/graphics/GameDisplay.java]: True, no idea how that's even there

File: src/main/java/solitaire/game/CardStack.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (Blackop778) [reply src/main/java/solitaire/game/CardStack.java]: But `cards` is empty at this point so `solveFlipType(flipType);` doesn't do anything.

[the remark this answers is not in the document]
Comment (nicholasmonaco) [reply src/main/java/solitaire/game/CardStack.java]: Oh that's true, I'll fix it in the next version

**Pull request merged**